src/ape_compile/_cli.py

```diff
@@ -42,6 +42,11 @@ def _include_dependencies_callback(ctx, param, value):
     help="Also compile dependencies",
     callback=_include_dependencies_callback,
 )
+@click.option(
```

Comment:
This may also be useful in the `ape pm compile` command. We could potentially add this to the `ape.cli.options` module so it can be re-used.